core/session.ts

```diff
@@ -570,7 +570,9 @@ export class Session {
   private checkTableConfigValidity(tables: dataform.ITable[]) {
     tables.forEach(table => {
       // type
-      if (!!table.type && !TableType.includes(table.type as TableType)) {
+      try {
+        setOrValidateTableEnumType(table);
+      } catch (e) {
```

Comment:
The error caught here could be from `are not equivalent` which would be misleading.

I think you can just remove this check altogether because `tableTypeStringToEnum` throws for it!

Reply:
Good point, replaced with checking `enumType` for  `UNKNOWN_TYPE` value.